pkg/sentry/pgalloc/pgalloc.go

```diff
@@ -579,7 +579,12 @@ func (f *MemoryFile) AllocateAndFill(length uint64, kind usage.MemoryKind, r saf
 	if err != nil {
 		return memmap.FileRange{}, err
 	}
-	dsts, err := f.MapInternal(fr, hostarch.Write)
+	access := hostarch.Write
+	if kind == usage.Tmpfs {
```

Comment:
And what about usage.Anonymous?

Reply:
Hmm...I don't think there is "populate" senario for usage.Anonymous